gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -142,6 +145,9 @@ func (driver *DriverRemoteConnection) Submit(traversalString string) (ResultSet,
 
 // submitBytecode sends a bytecode traversal to the server.
 func (driver *DriverRemoteConnection) submitBytecode(bytecode *bytecode) (ResultSet, error) {
+	if driver.closed {
+		return nil, errors.New("cannot invoke this method for closed connection")
```

Comment:
Will need to update this after error localization PR

Reply:
done in https://github.com/lyndonbauto/tinkerpop/pull/68/commits/402e323b1867ab4460f0bcd5f0a2515f6638366a